classes/plugins/load.py

```diff
@@ -0,0 +1,49 @@
+from multiprocessing import Array
+from . import api
+from . import event
+
+import os
+import logging
+import inspect
+
+def _getPlugins_() -> Array:
+    import pkgutil
```

Comment:
we would import this multiple times, is that intended?

Reply:
yes importing this part mutiple times is intended
if we dont want the *whole* plugins library we can just `from classes.plugin import load`